### ISSUE TITLE
Why ignore indentation inside arrays?

### ISSUE THREAD

**Issue by mkjois** [clutchski/coffeelint#583]
[This comment](https://github.com/clutchski/coffeelint/blob/master/src/rules/indentation.coffee#L61) provides a use case, but IMO people should just write it like this:
```
x = [
  "foo"
  "bar"
]
```

In my case, I found that coffeelint wasn't catching incorrect indentation for the following code (setting was at the default, 2 spaces):
```
angular.module 'myApp', [
    'ui.router'
]

.config [
    '$stateProvider'
    '$urlRouterProvider'
    ($stateProvider, $urlRouterProvider) ->
        $urlRouterProvider.otherwise '/'
        $stateProvider.state 'main',
            url: '/'
            template: '<div><p>Hello, world!</p></div>'
]
```
